ledger/ledger-api-test-tool/BUILD.bazel

```diff
@@ -28,6 +28,10 @@ load(
     ":conformance.bzl",
     "conformance_test",
 )
+load(
+    "@build_environment//:configuration.bzl",
+    "mvn_version",
+)
```

Comment:
I don't think this is needed anymore.

Reply:
Does this have to stay here?

Reply:
Sorry, missed it earlier, tried without it and it's indeed not needed.